src/plugins/shared_ux/public/components/solution_toolbar/items/button.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiButton } from '@elastic/eui';
+import { EuiButtonPropsForButton } from '@elastic/eui/src/components/button/button';
+
+import './button.scss';
+
+export interface Props
+  extends Pick<EuiButtonPropsForButton, 'onClick' | 'iconType' | 'iconSide' | 'className'> {
+  label: string;
+  primary?: boolean;
+  isDarkModeEnabled?: boolean;
```

Comment:
is this being used anywhere?

Reply:
Some of it is being used in the `solution_toolbar.tsx` file in the props. In the solution toolbar implementation, the primary styling is used consistently in how the toolbar is set up. 

I'm tempted to keep this as is since the items directory comes directly from the presentation team. I'm thinking as I migrate more components over, more and more references will occur with parts of the items folder.